cms/envs/aws.py

```diff
@@ -303,6 +303,19 @@
 ##### X-Frame-Options response header settings #####
 X_FRAME_OPTIONS = ENV_TOKENS.get('X_FRAME_OPTIONS', X_FRAME_OPTIONS)
 
+##### OAUTH2 Provider ##############
+if FEATURES.get('ENABLE_OAUTH2_PROVIDER'):
```

Comment:
This is similar to how it looks in the original `lms/envs/aws.py` config. This can probably be removed in both places without introducing issues, but since it's been like this in the lms config, I'd rather not risk introducing issues for other apps that might be checking against these flags albeit incorrectly.